### PR TITLE
[11.0][ADD] website_img_dimension

### PR DESCRIPTION
**Pull request by tarteo**
Simple module that (if you're an editor) shows the width and height (including original width and height) of
images on the website.

This module can be useful when you're e.g. optimizing your website.

image

### CODE REVIEW

**Comment by tarteo** [comment]
The error doesn't seem to be related to this module.

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
error is related to https://github.com/OCA/website/issues/487

File: website_img_dimension/static/src/js/website_img_dimension.tooltip.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            }).tooltip('show');
+        },
+        hide_tooltip: function($img) {
+            $img.tooltip('destroy');
+        }
+    });
+
+    var tooltip_manager = new TooltipManager();
+
+    return {
```
Comment (Daemo00) [inline website_img_dimension/static/src/js/website_img_dimension.tooltip.js:36]: Just for info, why are you returning both an instance and the Class itself?

**Review by Daemo00 — CHANGES_REQUESTED** [review]
Only code review, good in my opinion.

You should generate the README using https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py

File: website_img_dimension/static/src/js/website_img_dimension.tooltip.js (hunk 1 above)
Comment (tarteo) [reply website_img_dimension/static/src/js/website_img_dimension.tooltip.js:36]: I return the class so other modules can extend the TooltipManager, and I return the instance so other modules can invoke the functions or access variables of it. As TooltipManager is normally only created once like web.WebClient / web.web_client

**Comment by tarteo** [comment]
@Daemo00 I've generated the README. :+1:

**Comment by tarteo** [comment]
@Daemo00 Can you change your review, please?

**Review by Daemo00 — APPROVED** [review]
Ok for me

**Comment by tarteo** [comment]
Can this be merged? The travis error is not related to this module https://github.com/OCA/website/issues/487

**Comment by simahawk** [comment]
@tarteo there you go :)

**Pull request merged**